doc/source/quick_start.rst

```diff
@@ -282,8 +282,6 @@ Get file metadata
 Copy file
 =========
 
-Please switch to  :meth:`b2sdk.v2.Bucket.copy`.
```

Comment:
I couldn't import from `b2sdk.v2` and I didn't see any other references to a `v2` of the API so I've pulled this line out. Might this be for an upcoming release instead?

Reply:
it should be `b2sdk.v1.Bucket.copy` - v2 is not released yet and we don't need it any time soon

Reply:
Thank you. I've restored the reference and switched it to `v1`.